src/Engine/System/GameLogicSystem.hpp

```diff
@@ -2,15 +2,43 @@
 #ifndef _GAME_LOGIC_SYSTEM_HPP_
 #define _GAME_LOGIC_SYSTEM_HPP_
 
+
+
 #include "System.hpp"
+#include "Component/CameraComponents/CameraComponent.hpp"
+#include "Component/CameraComponents/CameraControllerComponent.hpp"
+#include "Component/PlayerComponents/PlayerControllerComponent.hpp"
+#include "Component/ImGuiComponents/ImGuiComponent.hpp"
+
 
-class GameLogicSystem : public System {
+
+// static class
+class GameLogicSystem {
```

Comment:
This doesn't need to derive System?

Reply:
Systems are static classes. There isn't a System parent anymore